source/includes/crud/Delete.java

```diff
@@ -1,8 +1,8 @@
-// Deletes multiple documents from a collection by using the Java driver
+// Deletes a document from a collection by using the Java driver
```

Comment:
This file now includes a deleteMany, so the old incorrect comment is now correct. Might be worth doublechecking these comments across files.